apps/src/templates/studioHomepages/SpecialAnnouncement.jsx

```diff
@@ -19,16 +19,12 @@ export default class SpecialAnnouncement extends Component {
 
   render() {
     const {isEnglish, isTeacher} = this.props;
-    const headingText = isEnglish
-      ? isTeacher
-        ? i18n.teacherAnnouncementSpecialWinter2021Heading()
-        : i18n.studentAnnouncementSpecial2023AiLaunchHeading()
-      : i18n.studentAnnouncementSpecial2023AiLaunchHeading(); // replaces International string
-    const descriptionText = isEnglish
-      ? isTeacher
-        ? i18n.teacherAnnouncementSpecialWinter2021Body()
-        : i18n.studentAnnouncementSpecial2023AiLaunchBody()
-      : i18n.studentAnnouncementSpecial2023AiLaunchBody(); // replaces International string
+    const headingText = isTeacher
+      ? i18n.teacherAnnouncementSpecialWinter2021Heading()
+      : i18n.studentAnnouncementHoc2023Heading();
+    const descriptionText = isTeacher
+      ? i18n.teacherAnnouncementSpecialWinter2021Body()
+      : i18n.studentAnnouncementHoc2023Body();
```

Comment:
Remove non-english string logic here since they should be the same regardless of language.